README.md

```diff
@@ -16,6 +16,9 @@ distributions.</p>
 ## Install
 
     sudo snap install helm
+    mkdir -p $HOME/snap/helm/common/kube
+    cp $HOME/.kube/config $HOME/snap/helm/common/kube
```

Comment:
Why doesn't the snap do this?

Reply:
The Snap does line 19 (`mkdir`), but only after the first time the `helm`command is run.  However, the initial command will fail in helm if the `config` file ins't present.

Reply:
Why not slurp up `$HOME/.kube/config` if and only if `$HOME/snap/helm/common/kube/config` does not exist? It seems odd to make every end user go through these steps manually.

```bash
mkdir -p $HOME/snap/helm/common/kube
if [ ! -e "$HOME/snap/helm/common/kube/config"] && [ -e "$HOME/.kube/config"]; then
  cp $HOME/.kube/config $HOME/snap/helm/common/kube/
else if [ ! -e "$HOME/snap/helm/common/kube/config"]; then
  echo "Please generate a Kubernetes config and copy it to $HOME/snap/helm/common/kube/config." 1>&2
  exit 1
fi
```

Reply:
I agree.  I would, ideally, like to point `$KUBECONFIG` to `$HOME/.kube/config` but this is blocked by snapd because it's a hidden directory (please correct me if I'm wrong)?

Just to confirm, is that snippet run post install?  Can you point to me to some docs as to how that works please?  Never done post install scripts.

Thanks!

Reply:
Actually, giving it a bit more thought, my suggestion will create confusion when you try and update your config in `$HOME/.kube/config/`. Perhaps this is best left as an explicit step. So I'd do something like this (untested):

helm.wrapper
```bash
#!/bin/sh -e
mkdir -p $HOME/snap/helm/common/kube
if [ ! -e "$HOME/snap/helm/common/kube/config"]; then
  echo "Please generate a Kubernetes config and copy it to $HOME/snap/helm/common/kube/config." 1>&2
  exit 1
fi

helm $@
```

snapcraft.yaml
```yaml
name: helm
version: '2.6.1'
summary: The Kubernetes package manager
description: |
  Helm is a tool for managing Kubernetes
  charts. Charts are packages of
  pre-configured Kubernetes resources.
grade: stable
confinement: strict

apps:
  helm:
    command: helm.wrapper
    plugs:
      - network
      - home

parts:
  wrapper:
    plugin: dump
    source: .
    stage:
      - helm.wrapper
  helm:
    plugin: dump
    source: https://storage.googleapis.com/kubernetes-helm/helm-v2.6.1-linux-amd64.tar.gz
```

Reply:
Yes, I was going to suggest something like this :)

Reply:
Pushed